include/abb_librws/rws_client.h

```diff
@@ -359,6 +359,13 @@ class RWSClient : public POCOClient
     logout();
   }
 
+  /**
+   * \brief Retrieves the controller service information.
```

Comment:
Is "controller service information" standard RWS nomenclature?

If it isn't, it would be good to provide a brief description of what it is.

Reply:
Clarified in 90ab7a9.

It is using the [controller](https://developercenter.robotstudio.com/api/rwsApi/ctrl_page.html) service of RWS to collect a list of controller resources. I generally try to use the terms and designations from the online documentation.